### PR TITLE
Update release notes with native-image info

### PR DESCRIPTION
**Pull request by Ibaqu**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 1]
```diff
@@ -82,6 +82,11 @@ To view bug fixes, see the [GitHub milestone for 2201.x.0 (Swan Lake)](https://g
 
 ### New features
 
+#### Native-image build (Experimental)
+Introduced the `--native` flag, which generates a GraalVM native image executable when building a ballerina project.
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:86]:
```suggestion
Introduced the `--native` flag, which generates a GraalVM native image executable when building a Ballerina project.
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 2]
```diff
@@ -82,6 +82,11 @@ To view bug fixes, see the [GitHub milestone for 2201.x.0 (Swan Lake)](https://g
 
 ### New features
 
+#### Native-image build (Experimental)
+Introduced the `--native` flag, which generates a GraalVM native image executable when building a Ballerina project.
```
Comment (azinneera) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:86]:
```suggestion
Introduced the `--native` flag, which generates a GraalVM native executable when building a Ballerina project.
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 3]
```diff
@@ -82,6 +82,11 @@ To view bug fixes, see the [GitHub milestone for 2201.x.0 (Swan Lake)](https://g
...
+- The generated executable contains the modules in the current package, their dependencies, Ballerina runtime, and statically linked native code from the JDK.
+- `$bal build --native`
```
Comment (azinneera) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:88]:
```suggestion
- `bal build --native`
```

**Review by Ibaqu — COMMENTED** [review]
@praneesha I added the ProjectAPI milestone issue board along next to the other milestone links from different areas but this section has been set under `Breaking changes`. 

Shouldnt this content move to a different section? Technically these arent `Breaking changes` right

**Comment by praneesha** [comment]
> @praneesha I added the ProjectAPI milestone issue board along next to the other milestone links from different areas but this section has been set under `Breaking changes`.
> 
> Shouldnt this content move to a different section? Technically these arent `Breaking changes` right

Please add the Project API bug fixes under that section. Breaking changes is a different section.

**Review by praneesha — APPROVED** [review]

**Review by azinneera — APPROVED** [review]

**Pull request merged**